trailing

```diff
@@ -0,0 +1,105 @@
+monin_obukhov/monin_obukhov.tech.ps:  vradius add translate hradius vradius scale 0 0 1 180 270 arc 
```

Comment:
What is this file?

Reply:
It is a postscript version created from the parsed html inline markup for documentation.  This is a merge of an old codebase and it may have re-introduced the html and PS files.  I'll make sure they are deleted prior to a PR merge.

Reply:
Master still contains these files.  I will delete them from this branch so they will be removed during the merge.

Reply:
Upon further review, the tech.ps document is a tech note on the equations and info about the parameterization.  Should we keep it?

Reply:
Thank you.

Reply:
Looks like we responded at roughly the same time.  

We should keep `monion_obukhov/monin_obukhov.tech.ps` (although I would rather it be a text/[la]tex document).  My question is about the `trailing` file.  It appears this may be something you accidentally added looking for trailing spaces in the repository.